test/release/examples/primers/modules.chpl

```diff
@@ -0,0 +1,390 @@
+/*
+ * Module Primer
+ *
+ * This primer introduces the concept of modules, a strategy for encapsulating
+ * code for later use.  It covers:
+ *   how to define a module
+ *   namespace control for a module's symbols
+ *   outside access of a module's symbols
+ *   namespace control when using a module in an outside context, including:
+ *     unlimited
+ *     explicit exclusion of symbols
+ *     explicit inclusion of symbols
+ *     renaming included symbols
+ *   outside access of modules in different files
+ */
+
+
+/* A module is a grouping of code - each program consists of at least one
+   module, and every symbol is associated with some module.  If all the code of
+   a file is not enclosed in an explicit module, defined using the syntax below,
+   then the file itself is treated as a module with the same name as the file
+   (minus the .chpl suffix).
+ */
+module modToUse {
+
+  /* In this case, foo is a public global variable which lives in the module
```

Comment:
I prefer to avoid "global" because I don't think of Chapel as really supporting global variables (i.e., 'foo' is not globally accessible, it's only accessible within modToUse and modules that use it).  "module-scoped"?  "module-level"?

"which" -> "that"

"lives in" -> "defined in" or "defined by"?